packages/@react-aria/tag/src/useTag.ts

```diff
@@ -82,7 +89,8 @@ export function useTag<T>(props: TagProps<T>, state: TagGroupState<T>, ref: RefO
     rowProps: {
       ...rowProps,
       tabIndex: (isFocused || state.selectionManager.focusedKey == null) ? 0 : -1,
-      onKeyDown: allowsRemoving ? onKeyDown : null
+      onKeyDown: allowsRemoving ? onKeyDown : null,
+      'aria-describedby': descProps['aria-describedby']
```

Comment:
VoiceOver appears not to read this at all...

Reply:
Reid and I discussed this briefly yesterday, the row's `aria-label` seems to be preventing the `aria-describedby` from announcing which is bizarre. Given how close we are to release and the need for translations, do we wanna punt on this here until after release? The behavior on VoiceOver prior to change is essentially the same, but the changes make it better for NVDA for sure (need to retest TalkBack with the modality check)